build_docs.py

```diff
@@ -294,21 +294,82 @@ class Language:
     def tag(self) -> str:
         return self.iso639_tag.replace("_", "-").lower()
 
-    @property
-    def is_translation(self) -> bool:
-        return self.tag != "en"
-
-    @property
-    def locale_repo_url(self) -> str:
-        return f"https://github.com/python/python-docs-{self.tag}.git"
-
     @property
     def switcher_label(self) -> str:
         if self.translated_name:
             return f"{self.name} | {self.translated_name}"
         return self.name
 
 
+@dataclasses.dataclass(frozen=True, kw_only=True, slots=True)
+class BuildMetadata:
+    _ver: Version
+    _lang: Language
```

Comment:
Let's use full names:
```suggestion
    _version: Version
    _language: Language
```